Makefile

```diff
@@ -63,17 +64,34 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
 .PHONY: fmt
-fmt: goimports ## Run goimports against code.
```

Comment:
Can we keep the `goimports` for now as we are using this in all the other projects as well.

Reply:
I will revert it,
however I followed the latest `kubebuilder` projects approach,
```
.PHONY: fmt
fmt: ## Run go fmt against code.
	go fmt ./... 
```